lib/MediaConnection.d.ts

```diff
@@ -0,0 +1,21 @@
+import * as EventEmitter from "events";
+
+type MediaConnectionType = 'NodeMediaConnection' | 'BrowserMediaConnection';
+
+export interface MediaConnection extends EventEmitter {
+    receiveSdpOffer(options: any): Promise<RTCSessionDescriptionInit>
```

Comment:
`createSdpOffer`

`RTCSessionDescriptionInit` is deprecated, use `RTCSessionDescription` instead.

Reply:
Changed options type to RTCOfferOptions and RTCSessionDescriptionInit to RTCSessionDescription